packages/ai-cli/tests/core.test.ts

```
@@ -0,0 +1,283 @@
+import {
+  bundledProviders,
+  factoryCandidatesForProvider,
+  resolveApiKey,
+  resolveModelSlug,
+} from '../src/core/providers'
+import { mergeOptions } from '../src/core/config'
+import { resolveOutputMode } from '../src/core/output'
+import { coerceFlags } from '../src/cli/options'
+import { CliError, ExitCode, toCliError } from '../src/core/exit-codes'
+import { inferMimeType, resolvePrompt } from '../src/core/io'
+import { join } from 'node:path'
```

ESLint

[error] `node:path` import should occur before import of `vitest`

(import/order)